src/algorithms/particle/CaloRemnantCombinerConfig.h

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2025 Subhadip Pal
+
+#pragma once
+
+#include <Evaluator/DD4hepUnits.h>
+
+namespace eicrecon {
+
+struct CaloRemnantCombinerConfig {
+
+  double deltaRAddEM = 0.03;
+  double deltaRAddH  = 0.15;
+};
```

Comment:
Just for easy reference, can't hurt to add a couple more docstrings!
```suggestion
  ///! radius around seed to add EMCal clusters
  double deltaRAddEM = 0.03;

  ///! radius around see to add HCal clusters
  double deltaRAddH  = 0.15;

};
```